docs/ingest-pipelines.asciidoc

```diff
@@ -1,21 +1,92 @@
-See <<apm-data-security,data security>> for an example.
+The default APM pipelines are defined in index templates that {fleet} loads into {es}.
+{es} then uses the index pattern in these index templates to match pipelines to APM data streams.
 
-[float]
-[[view-edit-default-pipelines]]
-=== View or edit a default pipeline
+[discrete]
+[id="default-pipeline{append-legacy}"]
+=== Default ingest pipelines
+
+// QQ: Do we want to doc these still?
+// Should we doc them on a per data-stream basis?
+
+The APM integration ships with a number of default ingest pipelines.
+
+[horizontal]
+Add ingest timestamp::
+Adds an ingest timestamp (`event.ingested`) to APM events.
+
+Add user agent::
+Adds `user_agent` information to APM events.
+
+Geo-IP data::
+Enriches APM data by adding user {ref}/geoip-processor.html[Geo-IP data] to the `client.geo` field.
+
+Remove span metadata::
+To save storage, metadata fields that are available on the parent transaction, like `host`,
+`kubernetes`, and `container`, are removed.
+
+Error grouping::
+Adds the `error.grouping_name` field to error documents.
+This groups similar errors together in the APM app.
+
+OpenTelemetry Metrics::
+Copies well-known OpenTelemetry metrics to their Elastic APM counterparts, enabling visualization in the APM app.
+For example, the OpenTelemetry metric field `runtime.jvm.gc.time` is copied to the Elastic APM metric field `jvm.gc.time`.
+Metrics are duplicated so you can refer to them by either the OpenTelemetry or Elastic APM metric name.
+
+[discrete]
+[id="view-edit-default-pipelines{append-legacy}"]
+=== View ingest pipelines
 
 To view or edit a default pipelines in {kib},
-select *Stack Management* / *Ingest Pipelines*.
+select **Stack Management** > **Ingest Pipelines**.
 Search for `apm`.
 
 See {ref}/ingest.html[ingest node pipelines] for more information.
 
-// to do: add more information including an example
\ No newline at end of file
+[float]
+[id="custom-pipelines{append-legacy}"]
+=== Add a custom ingest pipeline
+
+IMPORTANT: Fleet-managed index templates are overwritten on reinstall/upgrade.
+Any customizations to index pipelines are not persisted when upgrading.
+When editing {fleet}-managed index templates, ensure that the package-provided pipeline is still called.
+
+// tutorial would be something like this:
+. Find the pipeline **Stack Management** > **Index Management** > **Index Templates** > `traces-apm`.
+. **Settings** >
++
+[source,json]
+----
+{
+  "index": {
+    "default_pipeline": "traces-apm-{version}"
+  }
+}
+----
+
+// Edit default
+. **Stack Management** > **Ingest Pipelines** > Search for +traces-apm-{version}+.
+. **Manage** > **Edit** to edit the default pipeline.
+
+// Add custom pipeline
+. **Stack Management** > **Ingest Pipelines** > **Create pipeline**
+. Create it
+. **Stack Management** > **Ingest Pipelines** > Search for +traces-apm-{version}+.
+. **Manage** > **Edit** to edit the default pipeline. Add the new pipeline as the final step.
```

Comment:
Before I put time into creating this custom ingest pipeline tutorial... does this rough outline sound correct? I tested this flow and everything _seems_ to work.

Reply:
Sounds right, but I'm wondering if we should be documenting this. Not being able to edit pipelines in a way that persists is a known issue, but we don't know how much of a pain that is yet. By documenting the workaround so prominently we may not learn that. WDYT @simitt?

Reply:
Thanks. Removing from this PR for now (51b4dca). Added your thoughts to the pipeline doc issue: https://github.com/elastic/observability-docs/issues/1226